modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp

```diff
@@ -27,10 +27,12 @@ const std::vector<std::vector<size_t>> smoke_numSplits = {
     {4, 11, 6, 9},
 };
 
+const std::vector<size_t> smoke_axis = {0, 1, 2, 3};
+
 INSTANTIATE_TEST_CASE_P(smoke_NumSplitsCheck,
                         VariadicSplitLayerTest,
                         ::testing::Combine(::testing::ValuesIn(smoke_numSplits),
-                                           ::testing::Values(-3, -2, -1, 0, 1, 2, 3),
```

Comment:
Why did you removed negative values?

Reply:
Axis type in `VariadicSplitParams` is or `size_t`
https://github.com/openvinotoolkit/openvino/blob/288a750bc6f370b9b332cd118d14feb21657805c/src/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/variadic_split.hpp#L19

Reply:
> Why did you remove negative values?

It is have to be fixed in test. VariadicSplit supports negative axis https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/VariadicSplit_1.md
```
2: axis. Axis along data to split. A scalar or tensor with shape [1] of type T2 with value from range -rank(data) .. rank(data)-1. Negative values address dimensions from the end. Required.
```

Reply:
The link I provided is the general test class in the main Openvino repository.
Do you propose that we should fix it there?

Reply:
> The link I provided is the general test class in the main Openvino repository. Do you propose that we should fix it there?

Could you please check latest master? https://github.com/openvinotoolkit/openvino/pull/15925 was merged

Reply:
Updated it, thanks!